DistroLauncher/DistroLauncher.cpp

```diff
@@ -4,6 +4,7 @@
 //
 
 #include "stdafx.h"
+#include "AfterInstall.h"

```

Comment:
We can (and should) move those top level includes into the precompiled header (`stdafx.h`).

Reply:
Done!